docs/source/examples/Widget Asynchronous.ipynb

```diff
@@ -282,6 +282,213 @@
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Displaying a widget in the background\n",
+    "\n",
+    "While a widget's _value_ can be updated from a background thread, note that a widget cannot be (reliably) _displayed_ from a background thread.\n",
+    "\n",
+    "This is because output widgets are capturing based on the thread in which they were invoked. In other words, in the thread where the output is invoked, there is a context manager which starts and then stops the output capturing. If the threaded displays happen to occur during this time frame, they'll also be captured. However, if the threaded displays happen to occur after the main thread stops capturing output, those displays will not be put in that output widget.\n",
+    "\n",
+    "This means that sometimes, trying to display a widget from a background thread will appear to work fine, and sometimes one or more widgets that were supposed to be displayed will be simply missing."
```

Comment:
I think that saying users shouldn't try to display a widget from a background thread is important (I wasn't aware of this, so it's definitely useful). 

However, I don't think we should add an example that doesn't work. It just makes the documentation longer, and often, people copy and paste snippets directly from documentation without really reading the context.

Reply:
The key isn't so much on displaying a widget, it's that displaying *anything* may or may not be captured, depending on what's happening in the main thread.

Reply:
Hmm ok, I can delete this section. I was thinking that someone might just try threading and see if it works (which is what I did) and if they had started researching after that didn't work, it would be cool to see an explanation of what's happening. I'll try to refactor that into prose instead so that there is no nonworking code to copy/paste.